README.md

```diff
@@ -23,9 +23,9 @@ Automatically install the `extras` support dependencies for both USB and BLE.
 pip install godirect[usb,ble]
```

Comment:
From your email this morning, you mentioned that this was changing to "pip3 install godirect" - or did I misunderstand?

Reply:
Exactly. Good catch.